### ISSUE TITLE
Missing features in edition trace files

### ISSUE THREAD

**Issue by Lecrapouille** [apitrace/apitrace#597]
I may be wrong but it seems that it is not possible to remove a trace or move it to other position call.

In my case I'm debugging my OpenGL program and I noticed that, for example, I called many times glBindVertexArray(1). I would like to suppress one of them or change its called order position and check if the result is still correct.

I think this could be nice features.

**Comment by jrfonseca** [comment]
I agree.  Unfortunately, Apitrace hasn't been receiving much love in general, and the GUI in particular.

It's pretty labour intensive to add functionality to the GUI.  I feel that the ability of editing trace files via an ordinary text editor would be time better spent.

**Comment by Lecrapouille** [comment]
PS: just an observation concerning the GUI displaying matrices:
- I think it's not necessary important to display float with 16 digits after the comma, 3 is enough :)
- is it possible to add a carriage return (instead of ',') ?

This will save space because with a single line the GUI does not allow to display the whole matrix.

Maybe add a mouse click event for copying in the clipboard the matrix with the 16 digits and  its commas.
